proposals/lip-0063.md

```diff
@@ -469,31 +474,32 @@ def createValidatorsArray():
     validators = []
     validatorKeys = getValidatorKeys()
     for every account in accounts:
-        if account.pos.validator.username == "":
+        if account.dpos.delegate.username == "":
             continue
         validator = {}
         validator.address = account.address
-        validator.name = account.pos.validator.username
+        validator.name = account.dpos.delegate.username
         validator.blsKey = INVALID_BLS_KEY
         validator.proofOfPossession = DUMMY_PROOF_OF_POSSESSION
         if validatorKeys[account.address]:
             validator.generatorKey = validatorKeys[account.address]
         else:
             validator.generatorKey = INVALID_ED25519_KEY
-        validator.lastGeneratedHeight = account.pos.validator.lastForgedHeight
+        validator.lastGeneratedHeight = account.dpos.delegate.lastForgedHeight
         validator.isBanned = True
-        validator.reportMisbehaviorHeights = account.pos.validator.reportMisbehaviorHeights
```

Comment:
I think it should be `account.dpos.delegate.pomHeights` as this is the property name in v3, see https://github.com/LiskHQ/lips/blob/main/proposals/lip-0030.md.

Reply:
Right, fixed.